test/integration/targets/prepare_http_tests/tasks/kerberos.yml

```diff
@@ -40,7 +40,7 @@
     name:
     - decorator < 5.0.0 ; python_version < '3.5' # decorator 5.0.5 and later require python 3.5 or later
     - gssapi < 1.6.0 ; python_version <= '2.7' # gssapi 1.6.0 and later require python 3 or later
-    - gssapi ; python_version > '2.7'
+    - gssapi < 1.8.0 ; python_version > '2.7' # HACK: gssapi 1.8.0+ requires a newish pip with PEP517 support to build
```

Comment:
Wouldn't it better to put this in the constraint so that `gssapi` is constrained only on Python 2.7 and 3.6. This would be wherever `{{ remove_constraints }}` below is being picked up.

Reply:
cc @mattclay correct me if I'm wrong the `{{ remote_constraints }}` is just a copy of this file https://github.com/ansible/ansible/blob/devel/test/lib/ansible_test/_data/requirements/constraints.txt so adding the following line should work

```
gssapi < 1.8.0 ; python_version < '3.7' # Requires PEP 517 support which isn't present on RHEL 8 Python 3.6
```

Reply:
https://github.com/ansible/ansible/pull/78542